app/taskqueue/tasks/report.py

```diff
@@ -177,6 +178,41 @@ def send_bisect_report(report_data, email_opts, base_path=utils.BASE_PATH):
     return status
 
 
+@taskc.app.task(name="send-test-report")
+def send_test_report(job, git_branch, kernel, report_data, email_opts, base_path=utils.BASE_PATH):
+    """Send the test suite report email.
+
+    :param job: The job name.
+    :type job: string
+    :param git_branch: The git branch name.
+    :type git_branch: string
+    :param kernel: The kernel name.
+    :type kernel: string
+    :param report_data: The data necessary for generating a report.
+    :type report_data: dict
+    :param email_opts: Email options.
+    :type email_opts: dict
+    """
+    report_id = "-".join([job, git_branch, kernel])
+    utils.LOG.info("Sending test suite report email for '{}'".format(report_id))
+    status = "ERROR"
```

Comment:
Not strictly required as this gets initialised when calling `utils.emails.send_email()`.